bundles/AdminBundle/Resources/public/css/admin.css

```
@@ -1184,7 +1184,7 @@ span.warning {
 .pimcore_editable_wysiwyg {
     position:relative;
     padding: 10px;
-    min-height: 80px;
+    min-height: 100%;
```

I would suggest that we leave the minimum height at 80px. Otherwise, if no height is specified for the editor, as in the demo on the Car Class, the editor is quite small. 

```suggestion
    min-height: 80px;
    height: 100%;
```

---

That works as well, but I get a slight overflow which causes a scrollbar with that, on focus. Not sure if that id desirable.

---

No this is not desirable. But we should add some kind of minimum height, otherwise the WYSIWYG editor is too small.

---

I think this could do it https://caniuse.com/css-math-functions